source/DoubleCacheTests/DoubleCacheTests.cs

```diff
@@ -43,6 +43,42 @@ public void AddWithTimeout_CalledOnBoth()
         }
 
         [Fact]
+        public void Get_CalledOnLocal()
+        {
+            _doubleCache.Get("A", typeof(string), null);
+
+            A.CallTo(() => _local.Get("A", A<Type>.Ignored, A<Func<object>>.Ignored)).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => _remote.Get("A", A<Type>.Ignored, A<Func<object>>.Ignored)).MustNotHaveHappened();
+        }
+
+        [Fact]
+        public void Get_WithTimeToLive_CalledOnLocal()
+        {
+            _doubleCache.Get("A", typeof(string), null, TimeSpan.FromSeconds(1));
+
+            A.CallTo(() => _local.Get("A", A<Type>.Ignored, A<Func<object>>.Ignored, TimeSpan.FromSeconds(1))).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => _remote.Get("A", A<Type>.Ignored, A<Func<object>>.Ignored, TimeSpan.FromSeconds(1))).MustNotHaveHappened();
+        }
+
+        [Fact]
+        public void GetGeneric_CalledOnLocal()
+        {
+            _doubleCache.Get<string>("A", null);
+
+            A.CallTo(() => _local.Get("A", A<Func<string>>.Ignored)).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => _remote.Get("A", A<Func<string>>.Ignored)).MustNotHaveHappened();
+        }
+
+        [Fact]
+        public void GetGeneric_WithTimeToLive_CalledOnLocal()
+        {
+            _doubleCache.Get<string>("A", null, TimeSpan.FromSeconds(1));
+
+            A.CallTo(() => _local.Get("A", A<Func<string>>.Ignored, TimeSpan.FromSeconds(1))).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => _remote.Get("A", A<Func<string>>.Ignored)).MustNotHaveHappened();
+        }
+
+        [Fact]
```

Comment:
FYI All of the tests are identical to the async versions except that they call the synchronous methods.